style.css

```diff
@@ -0,0 +1,276 @@
+#header {
```

Comment:
I think it would be better to use Classes to style your html elements raher  than using a lot of id to each elemnt.